scripts/install

```diff
@@ -28,10 +29,10 @@ app=$YNH_APP_INSTANCE_NAME
 #=================================================
 # CHECK IF THE APP CAN BE INSTALLED WITH THESE ARGS
 #=================================================
-ynh_script_progression --message="Validating installation parameters..."
+ynh_script_progression --message="Validating installation parameters..." --weight=1
```

Comment:
--weight=1 is useless, isn't it ?

Reply:
Yes it's completely useless as: 
https://github.com/YunoHost/yunohost/blob/8ecd8fe6a334e578709120765468a5e8cb6ebd23/data/helpers.d/logging#L235

But added to be consistent with other `ynh_script_progression` lines